packages/adp-tooling/templates/project/webapp/manifest.appdescr_variant

```diff
@@ -7,6 +7,9 @@
   "namespace": "apps/<%- app.reference %>/appVariants/<%- app.id %>/",
   "version": "0.1.0",
   "content": [
+<%if(app.content){-%><% for(let change of app.content){-%>
+    <%- JSON.stringify(change, undefined, 4) %><%-","%>
```

Comment:
Are you sure it should be 4 instead of 2? Most prettier configs have space set to 2.

Reply:
yes, because it is an inner object